__tests__/app.test.js

```diff
@@ -58,20 +58,55 @@ describe("GET /api/articles/:article_id", () => {
             })
         })
     })
-    test("status: 404, responds with 'The request path does not exist' if the endpoint does not exist", () => {
+
+    test("status: 404, responds with 'Article not found', when the article ID does not exist", () => {
         return request(app)
-        .get("/api/artivles/1")
+        .get("/api/articles/99999")
         .expect(404)
         .then((res) => {
-            expect(res.body.msg).toBe('The request path does not exist')
+            expect(res.body.msg).toBe('Article not found')
         })
     })
-    test("status: 404, responds with 'Article not found', when the article ID does not exist", () => {
+})
+
+describe("GET /api/non-existing-endpoint", () => {
+    test("status 404: responds with 'The request path does not exist' if the endpoint does not exist", () => {
+    return request(app)
+    .get("/api/artivles/1")
+    .expect(404)
+    .then((res) => {
+        expect(res.body.msg).toBe('The request path does not exist')
+    })
+})
+    test("status 404: responds with 'The request path does not exist' if the endpoint does not exist", () => {
+    return request(app)
+    .get("/api/artibles")
+    .expect(404)
+    .then((res) => {
+        expect(res.body.msg).toBe('The request path does not exist')
+    })
+})
+})
+
+describe("GET /api/articles", () => {
+    test("status 200: responds with all articles with properties of author, title, article_id, topc, created_at, votes, article_img_url and comment_count", () => {
```

Comment:
I would just amend this test description to also say "...and responds with the articles sorted by date in descending order" 🙂